Clarifying vulnerability payment amounts and categories (#1756)
sourcegraph/about: handbook/engineering/security/reporting-vulnerabilities.md

### Bounties

> We provide monetary rewards, from $50 to $10,000 USD, for security vulnerability reports. The actual reward amount is determined based on the number of customers impacted, the difficulty of exploiting the vulnerability, and the severity of the consequences (e.g. service disruption, data leakage, reputational damage to Sourcegraph) of a successful exploit.

Reviewer: ```suggestion
We provide monetary rewards, from $50 to $10,000 USD, for security vulnerability reports. The actual reward amount is determined based on the number of customers impacted, the difficulty of exploiting the vulnerability, the severity of the consequences (e.g. service disruption, data leakage, reputational damage to Sourcegraph) of a successful exploit, and the quality of the security report.
```

When a monetary bounty is presented, eligible reports will be based on the severity, as determined by [CVSS v3.1](https://www.first.org/cvss/calculator/3.1). We will send payment to a valid PayPal account. We will ask you for the name and country associated with your PayPal account.

**Categories**

> |Attack Outcome| Maximum Payout|

Reviewer 2: ```suggestion
| Attack Outcome | Maximum Payout |
```

|:------------- | :----------: |
| You read or write to another user's code | $10,000 |

> | You take over another user's account  | $5000 |
> | You gain access to the internal Sourcegraph cloud network | $2500 |
> | You gain access to another user's configurations | $2000 |

Reviewer 2: Consistently use comma (or not)

```suggestion
| You take over another user's account  | $5,000 |
| You gain access to the internal Sourcegraph cloud network | $2,500 |
| You gain access to another user's configurations | $2,000 |
```
